src/components/identifier/identifier.njk

```diff
@@ -1,12 +1,84 @@
-<div class="padding-2">
-  <h4>Basic Identifier</h4>
-  {%- render '@identifier--basic' -%}
-</div>
-<div class="padding-2">
-  <h4>Multiple logos identifier</h4>
-    {%- render '@identifier--multiple-logos' -%}
-</div>
-<div class="padding-2">
-  <h4>No logos identifier</h4>
-  {%- render '@identifier--no-logos' -%}
+{% set taxpayer_disclaimer %}Produced and published at taxpayer expense.{% endset %}
+
+<div class="usa-identifier">
```

Comment:
Is there a specific reason for changing identifier from `<section> → <div>`?

Reply:
Wasn't sure if these met the definition of a section — specifically that they do not begin with a heading or might be considered "chapters"

Reply:
Voiceover with landmark browsing 
- Enable voiceover `cmd + F5`
- Enable rotor `ctrl + opt + u`
- Right/Left arrow to find landmarks

---

## Variations

### ❌ Current

![image](https://user-images.githubusercontent.com/3385219/92768198-68900b00-f35d-11ea-809f-28ace6f3c1c4.png)

### ❌ Div w/ aria-label
![image](https://user-images.githubusercontent.com/3385219/92768506-b147c400-f35d-11ea-892d-060f41568ff5.png)

### ✅ Section w/ aria-label
![image](https://user-images.githubusercontent.com/3385219/92768734-e9e79d80-f35d-11ea-852c-5d00e4668b43.png)

Reply:
I now see three areas with the new section updates:
![image](https://user-images.githubusercontent.com/3385219/92777818-3040fa80-f366-11ea-9140-cc8a2a2a0803.png)

Reply:
OK — I think that's where we want it.